api/provider/aws/defaults.go

```diff
@@ -1,5 +1,7 @@
 package aws
 
+import "github.com/quintilesims/layer0/common/models"
+
 const (
 	DEFAULT_INSTANCE_SIZE  = "m3.medium"
```

Comment:
This issue was originally brought on by me - I don't think Go uses all caps for constants. I've just seen upper-camel case